plugin/pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -24,19 +24,35 @@ const (
 	memoryResoureceName  string = "Memory"
 )
 
+var (
+	// The predicateName tries to be consistent as the predicate name used in DefaultAlgorithmProvider defined in
+	// defaults.go (which tend to be stable for backward compatibility)
+	ErrDiskConflict              = newPredicateFailureError("NoDiskConflict")
```

Comment:
I think it's better as-is; as his comment says, he is reusing the predicate names from the code. All of the names we already have are pretty inconsistent (positive vs. negative sense, etc.) but I don't think it's wroth trying to fix here.